rules.md

```diff
@@ -70,7 +72,7 @@ Each **Role** has **Skills** and **Limitations** alongside a **Personal Objectiv
 
 ### Restricting **Access**
 
-When a player **Desiminates** a card they MUST choose if that card is **Open Access**, and hence available for all players to use, or if the card is **Restricted Access**, and hence only accessible exclusively to them. Tokens are placed on cards to denote the accessibility of the card, where a **Green Token** indicates a card is **Open Access**, and an **Orange Token** indicates that a card is **Restricted Access**
+When a player **Desiminates** a card they MUST choose if that card is **Open Access**, and hence available for all players to use, or if the card is **Restricted Access**, and hence only accessible exclusively to them. Tokens are placed on cards to denote the accessibility of the card, where a green token indicates a card is **Open Access**, and an orange token indicates that a card is **Restricted Access**.
```

Comment:
We should stick to a single terminology for the tokens, I'm not that fused which but you've used 'Open Access Token' and 'Restricted Access Token' above and you should use that here - and throughout the document - if thats what we want to settle on. Sure it seems a little verbose, but it's a rule book.

Reply:
👍 for 'Open Access Token' and 'Restricted Access Token'. Removing mention of colour means these tokens can be anything that's to hand.

Reply:
Yeah, I played with someone who is color blind and they really struggled with the orange and green skittles!